tests/python_backend/test_graph_compilation.py

```diff
@@ -0,0 +1,277 @@
+import pytest
+
+from myia.compile.backends.python.python import compile_graph
+from myia.parser import parse
+from myia.utils.info import enable_debug
+
+
+def parse_and_compile(function):
+    with enable_debug():
+        graph = parse(function)
+    # from myia.ir.print import str_graph
+    # print(str_graph(graph))
```

Comment:
```suggestion
```

Or add a flag to toggle the printing, if you think that's useful to keep for debugging purposes.